components/screens/RegisterScreen.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import {
+    View,
+    StyleSheet,
+    Text,
+    TouchableOpacity
+} from 'react-native';
+
+import Icon from "react-native-vector-icons/FontAwesome";
+
+import { useNavigation } from '@react-navigation/native';
+
+import CustomButton from '../reusable/CustomButton';
+import CustomInput from '../reusable/CustomInput';
+
+function RegisterScreen(props) {
+    const [password, setPassword] = useState("")
+    const [isHidden, setIsHidden] = useState(true)
+
+    const navigation = useNavigation();
+
+    const [isPasswordFocused, setIsPasswordFocused] = useState(false);
+    const [isLoginFocused, setIsLoginFocused] = useState(false);
+
+    const handleShowPassword = () => {
+        if (password.length === 0) {
+            return;
+        }
+        setIsHidden(!isHidden)
+    }
+
+    const handleRegister = () => {
+        navigation.navigate("Dashboard");
+    }
+
+    return (
+        <View style={styles.container}>
+            <View>
+                <TouchableOpacity style={styles.loginButton}>
+                    <Icon
+                        name="google"
+                        size={15}
+                        color="red"
+                        style={styles.loginBsuttonIcons}
+                    />
+                    <View style={styles.LoginButtonTextContainer}>
+                        <Text style={styles.loginButtonText}>
+                            Continue with Google
```

Comment:
you can see here the same code are pasted from the other screen you might have this as component and put it there